week8/src/Home.js

```diff
@@ -29,22 +43,37 @@ const apiError =()=>{
 const searchMovie = async(e)=>{
     e.preventDefault();
     try{
-const url = `https://api.themoviedb.org/3/search/movie?api_key=b7d68d57b175ae831b45672648c74d7b&query=${title}`
+
 const res=await fetch(url);
 const data = await res.json();
 setMovies(data.results)
 if(data.total_results==0){
     notFound();
+}
+if(data.results==0){
+    noPages();
 }
     }catch(e){
    apiError();
     }
 }
+function saveUserSearch(){
```

Comment:
We are storing all user searches and only showing 5. can we just only store the most recent 5?